File: Domain/Entities/Persons/Steward.cs
using System;

namespace Domain.Entities.Persons
{
    public class Steward
    {
        public Steward()
        {
            Assignment = new Assignment();
        }
        
        public Guid StewardId { get; set; }
        public virtual GameSession GameSession { get; set; }
        public Guid GameSessionId { get; set; }
        public string Name { get; set; } = "";
        public int Skill { get; set; }
        public int Resources { get; set; }
        public int Loyalty { get; set; }
        public int Age { get; set; }
        public virtual Assignment Assignment { get; set; }
        public Guid AssignmentId {get; set;}
    }
}

File: reactui/src/common/models/steward.ts
export interface ISteward {
    stewardId: string;
    name: string;
    age: number;
    skill: number;
    loyalty: number;
    resources: number;
    industryId: string | null;
    developmentId: string | null;
    fiefId: string | null;
    marketId: string | null;
    portId: string | null;
    shipyardId: string | null;
}
